subsys/bluetooth/audio/mcc_internal.h

```diff
@@ -0,0 +1,113 @@
+/** @file
+ *  @brief Internal APIs for Bluetooth MCP.
+ */
+
+/*
+ * Copyright (c) 2020 Bose Corporation
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```

Comment:
What are these based from? The file that the majority of the content is copied from seems to have `Copyright (c) 2019 - 2021 Nordic Semiconductor ASA`

Reply:
Must have copied something else by accident, I corrected it